### DISCUSSION TITLE
default populations contradiction

### DISCUSSION THREAD

**Discussion by RubenBies** [MilesCranmer/PySR#327] [General]
Hey! On your page it says the default populations is 15 under 'Reference' but under 'features and options' it says its 100. I have been running some searches with 15 and 24 since you also said to set populations = 3 x num_cores. I wanted to try running more populations since i was confused by the contradiction on your page and my script crashed when trying to search 10k samples (only 2 features though) I assume its a Ram problem but wanted to ask anyway if there's a way to optimize my configuration for more populations than is good for me. Sorry if that's stupid.

Thank you so much for doing all this work, i am currently writing my bachelors thesis and having a blast using PySR!

**Comment by MilesCranmer** [comment]
Oops, thanks for pointing that issue out!! I will fix it.

**Comment by MilesCranmer** [comment]
> I wanted to try running more populations since i was confused by the contradiction on your page and my script crashed when trying to search 10k samples (only 2 features though) I assume its a Ram problem but wanted to ask anyway if there's a way to optimize my configuration for more populations than is good for me. Sorry if that's stupid.

Hm, this sounds like it might be a bug. Could you post an issue with the log and specific script you are using?

**Comment by RubenBies** [comment]
Thanks, I will try to write an issue but don't have much to go on.
